requirements/edx/github.txt

```diff
@@ -58,7 +58,7 @@ git+https://github.com/edx/ecommerce-api-client.git@1.1.0#egg=ecommerce-api-clie
 -e git+https://github.com/edx/edx-user-state-client.git@30c0ad4b9f57f8d48d6943eb585ec8a9205f4469#egg=edx-user-state-client
 -e git+https://github.com/edx/edx-organizations.git@release-2015-08-03#egg=edx-organizations
 
-git+https://github.com/edx/edx-proctoring.git@release-2015-08-18#egg=edx-proctoring==0.6.0
+git+https://github.com/edx/edx-proctoring.git@7d3e29a6fdb61d06dd30c5958f4504440979e40f#egg=edx-proctoring==0.6.1
```

Comment:
I presume this was just for your testing and doesn't want to go into this change.